cmd/selfupdate/selfupdate.go

```diff
@@ -169,7 +169,7 @@ func InstallUpdate(ctx context.Context, opt *Options) error {
 		} else {
 			err := installPackage(ctx, opt.Beta, newVersion, siteURL, opt.Package)
 			if err == nil {
-				fmt.Printf("Successfully updated rclone package from version %s to version %s\n", oldVersion, newVersion)
+				fs.Infof(nil, "Successfully updated rclone package from version %s to version %s", oldVersion, newVersion)
```

Comment:
Info is only shown when `-v` flag is present.
I think this message should be printed with `fs.Logf` (aka `NOTICE`) so it's visible by default. I'm just not sure that `--quiet` will hide it.

Reply:
@yedamao
Could please check that `--quiet` will hide the `fs.Logf` messages?

Reply:
--quiet/-q will set LogLevvel as LogLevelError, so fs.Logf is ok.

Reply:
Please use Logf here.

Reply:
this diff is outdated, please check the latest diff.

Reply:
ah I see now